app/src/main/java/com/example/movopfy/features/home/data/repository/AnilibriaRepositoryImpl.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.movopfy.features.home.data.repository
+
+import com.example.movopfy.features.home.data.api.anilibria.Anilibria
+import com.example.movopfy.features.home.data.models.AnilibriaSchedule
+import com.example.movopfy.features.home.domain.repository.AnilibriaRepository
+
+class AnilibriaRepositoryImpl(private val anilibria: Anilibria) : AnilibriaRepository{
+    override suspend fun getSchedule(): List<AnilibriaSchedule> {
+        return anilibria.getSchedule()
+    }
+
```

Comment:
please remove empty lines all over the project

Reply:
I did